app.js

```diff
@@ -8,7 +8,7 @@ require('./app_api/models/db');
 var uglifyJs = require("uglify-js");
 var fs = require('fs');
 
-var routes = require('./app_server/routes/index');
+// var routes = require('./app_server/routes/index');
```

Comment:
@marcgardiner why commented? any fair reason? why not remove?

Reply:
@igortb84 , that should be removed.

Reply:
ok, just removed it